package.json

```diff
@@ -1,17 +1,12 @@
 {
   "name": "rsdoctor-monorepo",
   "private": true,
-  "workspaces": {
```

Comment:
Why delete this?

Reply:
we use `pnpm-workspace.yaml`, i think it is enough

Reply:
changeset will read this field, removing it may break changeset

Reply:
fixed